frontend/src/component/releases/ReleasePlanTemplate/TemplateForm/TemplateForm.tsx

```diff
@@ -81,6 +83,11 @@ export const TemplateForm: React.FC<ITemplateFormProps> = ({
             description={<TemplateFormDescription />}
             formatApiCode={formatApiCode}
         >
+            {archived && (
+                <Alert severity='warning'>
+                    This template has been archived and can no longer be edited.
+                </Alert>
+            )}
```

Comment:
Should we also have this condition on the save button?

Reply:
Thank you, yes it's in EditTemplate where the button is passed in